### ISSUE TITLE
Redundant attributes in modal's built-in close button

### ISSUE THREAD

**Issue by phenderson00** [Esri/calcite-components#7125]
### Actual Behavior

The `CalciteModal`'s built-in close button has both of the following attributes: `title` and `aria-label`. Both attributes are being set to the string "Close". Setting both attributes to the same value is redundant.

### Expected Behavior

Either the `title` or `aria-label` attribute should be removed in order to eliminate redundancy. If the intention is to display a tool tip on hover, the `aria-label` can be removed.

### Reproduction Sample

https://codesandbox.io/s/calcite-modal-button-a11y-bug-s2lsiz?file=/src/App.js

### Reproduction Steps

1. Open the DevTools console
2. Click the Launch Modal button
3. Use the element selector to select the modal's close (X) button
4. See that the `title` and `aria-label` attributes on the button are both set to "Close"

### Reproduction Version

1.4.2

### Relevant Info

Windows 10, Chrome 114.0.5735.90

### Priority impact

p4 - not time sensitive

### Impact

This does not block our project, but the accessibility tester on the customer's side is the one who brought this issue to our attention.

### Esri team

Professional Services - Midwest Delivery Center

**Comment by driskull** [comment]
I'm not sure its redundant since the title is for the tooltip and the aria-label is for screen readers. Is it repeating the same text in a screen reader?

**Comment by driskull** [comment]
Gotcha. thanks! We can remove the title attribute I think.

**Comment by geospatialem** [comment]
> We're hearing the same text being read twice by the screenreader.

Was able to repro with NVDA across browsers - both JAWS and VoiceOver are not affected. This doesn't seem to be related to the `aria-label` and `title`, but rather the `dialog` role with NVDA mentioned in an issue on their repo at https://github.com/nvaccess/nvda/issues/8971.

The issue highlights [a sample](http://www.3needs.org/en/testing/code/role-dialog-code.html) showcasing the behavior in NVDA.

**Comment by geospatialem** [comment]
Closing this issue as not planned, as it pertains to a NVDA bug. re: https://github.com/Esri/calcite-components/issues/7125#issuecomment-1581545700.

For additional context, previously the `title` and `aria-label` sequence [was an issue with NVDA](https://github.com/nvaccess/nvda/issues/7841), but was [mitigated in September 2021](https://github.com/nvaccess/nvda/pull/12888).
